pyinstl/instlMisc.py

```diff
@@ -10,6 +10,10 @@
 import time
 
 import utils
+
+# GuyC: not sure why cannot do import multi_file as it resides near utils.py so instead we do:
```

Comment:
No need for any special import for MultiFileReader, just call it as utils.MultiFileReader